src/ramme/addRammeHandler.test.ts

```diff
@@ -0,0 +1,34 @@
+import { mockRepoFactory } from '../core/test.fixture'
+import { addRammeHandler } from './addRammeHandler'
+
+describe('addRammeHandler', () => {
+  let req: any
+  let res: any
+  let repo: any
+
+  const addHandler = addRammeHandler
+
+  beforeEach(() => {
+    jest.resetAllMocks()
+    repo = mockRepoFactory()
+    res = {
+      send: jest.fn(),
+      status: jest.fn(),
+    }
+    req = {
+      body: {
+        user_name: 'Florida Man',
+        text: undefined,
+      },
+    }
+  })
+
+  describe('happy case', () => {
+    it('should save ramme', async () => {
+      expect.assertions(1)
+      req.body.text = 'add testpass'
+      await addHandler(req, res, repo)
+      expect(repo.save).toHaveBeenCalled()

```

Comment:
This test does actually not test anything more then that save() has been called. I personally think it's more important to test that the input corresponds to what have been saved :)

Reply:
I agree. I will add more tests later. It will be easier now that the test fixture exists.